### PR TITLE
docs: add Nishu Goel to GDE list

### PR DESCRIPTION
**Pull request by NishuGoel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by petebacondarwin** [comment]
Hi Nishu,
Can you add an actual image file - currently the file you uploaded only contains text.

**Comment by AndrewKushnir** [comment]
@NishuGoel thanks for updating the PR and adding the image.

The CI indicated that the image is over 30kb limit, could you please resize it?

Also CI indicated that the commit messages are missing the scope part. Please merge all commits into one and update commit message to include `docs: ` in front of the message, for example:

```
docs: add Nishu Goel to GDE list
```

Thank you.

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -677,6 +677,14 @@
     "bio": "Nir is a Principal Frontend Consultant & Head of the Angular department at 500Tech, Google Developer Expert and community leader. He organizes the largest Angular meetup group in Israel (Angular-IL), talks and teaches about front-end technologies around the world. He is also the author of two books about Angular and the founder of the 'Frontend Band'.",
     "groups": ["GDE"]
   },
+  "nishugoel": {
+    "name": "Nishu Goel",
+    "picture": "nishugoel.jpg",
+    "twitter": "TheNishuGoel",
+    "website": "https://www.unravelweb.dev",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json:684]: This URL doesn't seem to work. Did you mean:

```suggestion
    "website": "https://unravelweb.dev",
```

**Comment by mgechev** [comment]
@NishuGoel you may have to squash your commits.

**Review by mgechev — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
Hi @NishuGoel, this PR was approved. Could you please merge all commits into a single one (that has the "docs: add Nishu Goel to GDE list" commit message), so that we can proceed with merging this PR? Thank you.

**Comment by AndrewKushnir** [comment]
@NishuGoel FYI I've rebased the PR and merged all commits into one. Here is a preview of the Angular.io website page:

https://pr41347-b8a7d44.ngbuilds.io/about?group=GDE

Please let us know if you want to update the image or we can proceed with merging this PR keeping the current picture.

Thank you.

**Comment by NishuGoel** [comment]
@AndrewKushnir Thanks. I have updated the image for a better preview on the Angular.io website page. Please check

**Pull request closed without merging**